### PR TITLE
Interoperability resources 

### PR DESCRIPTION
**Pull request by emdann**
This is a draft page providing pointers to resources for interoperability between scverse data structures and programming languages different from python, extending the current [anndata interoperability page](https://anndata.readthedocs.io/en/latest/interoperability.html). Since the single-cell best practices book will have an [extensive tutorial on interoperability](https://www.sc-best-practices.org/introduction/interoperability.html), the general consensus was to link this resource on top, and provide here links to external resources.

For the R/python object conversions I tried to specify whether the linked resource works in R or python and if it allows conversion within a session or if it does conversion through reading/writing other files.

### Outstanding issues

- Do we list everything or only recommended/functional resources? In my experience conversions can go wrong in really unpredictable ways, so having several options is often useful, but certain options out there are really broken (e.g. SeuratDisk, hence there is no mention)
- Should we align this and the interoperability page in anndata docs? @ivirshup

### CODE REVIEW

**Comment by grst** [comment]
Hi @emdann, 

I think you need to allow maintainers to push to your branch, otherwise the CI cannot add its changes.

**Comment by emdann** [comment]
I think it's on 
Screenshot 2022-12-09 at 15 26 25

**Comment by grst** [comment]
hmm...

**Comment by grst** [comment]
this one has worked :) 
https://scverse-tutorials--32.org.readthedocs.build/en/32/notebooks/scverse_data_interoperability.html

**Comment by emdann** [comment]
Yay! The JavaScript title is badly rendered but looks great otherwise

File: docs/notebooks/scverse_data_interoperability.ipynb [hunk 1]
```diff
@@ -0,0 +1,139 @@
+{
```
Comment (grst) [inline docs/notebooks/scverse_data_interoperability.ipynb:1]: <p>Loom is actually implemented in AnnData (scanpy just re-exports that afaik), and there's also a <code>write_loom</code> function.  </p><p> </p><p>https://anndata.readthedocs.io/en/latest/api.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (grst) [inline docs/notebooks/scverse_data_interoperability.ipynb:1]: <p>As noted before, the javascript headline is not rendered correctly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by grst** [comment]
This is excellent! Let's just fix the javascript header and merge this.

**Comment by emdann** [comment]
Should be fixed now, ready to merge!

**Pull request merged**